sample/perf/index.html

```diff
@@ -69,30 +79,39 @@
     const median = times[Math.floor(n / 2)];
     const out = [
       name,
-      `Median: ${median}, `,
+      `Median: ${median}`,
     ];
     if (opts.size) {
       out.push('Throughput: ' + humanSize(opts.size / median * 1000) + '/s');
     }
-    console.log(...out);
+    log(...out);
   }
 
-  async function main() {
-    console.log('Running benchmarks, please wait...')
+  window.main = async function (invoker) {
+    log(`Running benchmarks with ${invoker.constructor.name} please wait...`)
+    await benchmark({ size: 1024 * 1000 },
+      (bench) => benchmark_populate(invoker, bench, { numKeys: 1000, parallel: false }));
     await benchmark({ size: 1024 * 1000 },
-      (bench) => benchmark_populate(bench, { numKeys: 1000, parallel: false }));
+      (bench) => benchmark_populate(invoker, bench, { numKeys: 1000, parallel: true }));
     await benchmark({ size: 1024 * 1000 },
-      (bench) => benchmark_populate(bench, { numKeys: 1000, parallel: true }));
-    console.log('Done!');
+      (bench) => benchmark_scan(invoker, bench, { numKeys: 1000 }));
+    log('Done!');
   }
 
   function humanSize(bytes) {
-    if (bytes == 0) { return "0.00 B"; }
+    if (bytes == 0) { return '0.00 B'; }
     var e = Math.floor(Math.log(bytes) / Math.log(1024));
     return (bytes / Math.pow(1024, e)).toFixed(2) + ' ' + ' KMGTP'.charAt(e) + 'B';
   }
 
-  main();
+  function log() {
+    document.querySelector('#output').value += Array.prototype.join.call(arguments, ' | ') + '\n';
```

Comment:
Why u no rest parameters?

Reply:
Just hard to keep track of all the sugar in these languages.

Reply:
How is `Arrray.prototype.join.call(arguments)` easier to keep track of? I guess old habits die hard =P

Reply:
I think I have a deep seated memory that `arguments` is weird and nothing you'd expect will work with it.

Reply:
That is correct. arguments is never needed in ES6. We killed it!